test/appium/tests/base_test_case.py

```diff
@@ -30,7 +30,7 @@ def sauce_access_key(self):
 
     @property
     def executor_sauce_lab(self):
-        return 'https://%s:%s@ondemand.eu-central-1.saucelabs.com:443/wd/hub' % (self.sauce_username, self.sauce_access_key)
+        return 'http://%s:%s@ondemand.eu-central-1.saucelabs.com:80/wd/hub' % (self.sauce_username, self.sauce_access_key)
```

Comment:
Why tho? HTTP is insecure. Is it failing with HTTPS?

Reply:
Just tried if 433 was the reason of why many tests are failing after that migration (we had it previously on 80th). For example https://ci.status.im/job/end-to-end-tests/job/status-app-prs/1575/console ``` HTTPSConnectionPool(host='ondemand.eu-central-1.saucelabs.com', port=443): Max retries exceeded ```

Reply:
But the issue appears to not be `Max retries exceeded` but `too many redirects`:
```
urllib3.exceptions.MaxRetryError:
HTTPSConnectionPool(host='ondemand.eu-central-1.saucelabs.com', port=443):
Max retries exceeded with url: https://****:****@ondemand.eu-central-1.saucelabs.com:443/wd/hub/session/2a95c0353773469e9b58b3917d0033d6?ref=ed95194737c64f728cfb5ddd9b23dc00 
(Caused by ResponseError('too many redirects'))
```
Have you tried not specifying the port?

Reply:
A regular `GET` request works just fine:
```
 > curl -s 'https://ondemand.eu-central-1.saucelabs.com/' 
{"ready":true,"message":"Sauce Labs automated testing is fully operationa
```

Reply:
Yes but it didn't help without port - removed in https://github.com/status-im/status-react/pull/11628/commits/c9b40ab32da9814fa167a7b1051240217de73a8a (but used http there though). Some tests are still failing https://ci.status.im/job/end-to-end-tests/job/status-app-prs-rerun/481/console with this 'too many redirects'.

Reply:
They are not all ERRORing with it, - but those which are failed screw all further threads of tests...

Reply:
Makes sense. I guess we're fine with HTTP for now, but could we open some kind of Support ticket with SauceLabs to ask about this issue?

Reply:
Already reached their support

Reply:
Sweet.